dynamic_flag/front.py

```diff
@@ -32,6 +34,8 @@
 # shm_exec sets /dev/shm no longer be noexec. Default = keep noexec
 shm_exec = 1 if os.environ.get("hackergame_shm_exec") == "1" else 0
 
+lxcfs_opts = '-v /var/lib/lxcfs/proc/cpuinfo:/proc/cpuinfo:rw -v /var/lib/lxcfs/proc/diskstats:/proc/diskstats:rw -v /var/lib/lxcfs/proc/meminfo:/proc/meminfo:rw -v /var/lib/lxcfs/proc/stat:/proc/stat:rw -v /var/lib/lxcfs/proc/swaps:/proc/swaps:rw -v /var/lib/lxcfs/proc/uptime:/proc/uptime:rw -v /var/lib/lxcfs/proc/slabinfo:/proc/slabinfo:rw -v /var/lib/lxcfs/sys/devices/system/cpu:/sys/devices/system/cpu:rw'
```

Comment:
Why not set them to `ro`? It looks like that they are not meant to be written inside container.

Reply:
It's in the README of lxcfs: https://github.com/lxc/lxcfs?tab=readme-ov-file#using-with-docker

Reply:
Well, I would prefer `ro` if possible -- allow containers writing to them would possibly increase the attack surface.